README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="art/Logotype/Logotype512.png" alt="Proje ismi" height="60px"></p>
```

Comment:
Was "Proje ismi" supposed to mean something?

Reply:
I'm fixed this. Project Name will be written there.